demo/kserve/deploy-remove.md

```diff
@@ -123,151 +97,151 @@ Note: The **flan-t5-small** LLM model has been containerized into an S3 MinIO bu
-      ```
-
-      The response should be similar to the following:
-
-      ```json
-      {
-        "details": {
-
-        }
-      },
-      {
-        "tokens": [
-          {
-            "text": "▁",
-            "logprob": -1.599083423614502
-          }
-        ],
-        "details": {
-          "generated_tokens": 1
-        }
-      }
-      {
-        "generated_text": "74",
-        "tokens": [
-          {
-            "text": "74",
-            "logprob": -3.3622500896453857
-          }
-        ],
-        "details": {
-          "generated_tokens": 2
-        }
-      }
-      ....
-      ```
-
-
-    3-gRPC. Perform inference with Remote Procedure Call (gPRC) commands. This example uses the [`grpcurl`](https://github.com/fullstorydev/grpcurl) command-line utility.
-
-      a. Determine whether the HTTP2 protocol is enabled in the cluster.
-
-      ```bash
-      oc get ingresses.config/cluster -ojson | grep ingress.operator.openshift.io/default-enable-http2
-      ```
-
-      If the annotation is set to true, skip to Step 3c.
-
-      b. If the annotation is set to either false or not present, enable it.
-
-      ```bash
-      oc annotate ingresses.config/cluster ingress.operator.openshift.io/default-enable-http2=true
-      ```
-
-      c. Run the following `grpcurl` command for all tokens in a single call:
-
-      ```bash
-      grpcurl -insecure -d '{"text": "At what temperature does liquid Nitrogen boil?"}' -H "mm-model-id: flan-t5-small-caikit" ${KSVC_HOSTNAME}:443 caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict
-      ```
-
-      The response should be similar to the following:
-
-      ```json
-      {
-        "generated_token_count": "5",
-        "text": "74 degrees F",
-        "stop_reason": "EOS_TOKEN",
-        "producer_id": {
-          "name": "Text Generation",
-          "version": "0.1.0"
-        }
-      }
-      ```
-
-      d. Run `grpcurl` to generate a token stream.
-
-      ```bash
-      grpcurl -insecure -d '{"text": "At what temperature does liquid Nitrogen boil?"}' -H "mm-model-id: flan-t5-small-caikit" ${KSVC_HOSTNAME}:443 caikit.runtime.Nlp.NlpService/ServerStreamingTextGenerationTaskPredict
-      ```
-
-      The response should be similar to the following:
-
-      ```json
-      {
-        "details": {
-
-        }
-      },
-      {
-        "tokens": [
-          {
-            "text": "▁",
-            "logprob": -1.599083423614502
-          }
-        ],
-        "details": {
-          "generated_tokens": 1
-        }
-      }
-      {
-        "generated_text": "74",
-        "tokens": [
-          {
-            "text": "74",
-            "logprob": -3.3622500896453857
-          }
-        ],
-        "details": {
-          "generated_tokens": 2
-        }
-      }
-      ....
-      ```
+   - http only. Perform inference with HTTP. This example uses cURL.
```

Comment:
Apart from this line, this block hasn't really changed, only the level of indentation has